### ISSUE TITLE
Screen reader content order is wrong (iOS only)

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#564]
### Issue (iOS only)
On screen 1, the third block of text (starting with “With BC wallet…) gets skipped by the screen reader.

Screens 2, 3 and 4 currently read out only the headings and skip all other text, then repeat in sequence by reading out all the skipped text from screens 2, 3 and 4 without the headings, then the skipped third block from screen 1, then “Get started” (button text), “back,” and “next.”

The reading order for the biometrics page appears to be incorrect. It starts on a blank paragraph between the first and second paragraphs.

### Fix
Check the semantic structure of these screens on iOS. There’s a sequencing issue in terms of how the reading order is currently being interpreted by the screen reader. Ideally, it should be:
• Title
• Headings
• Body text
• Navigation as appropriate depending on where it is expected to be used in the context of the user journey

*This order should be used throughout the entire app

**Comment by jleach** [comment]
@knguyenBC Still an issue? I seem to recall fixing some order issues you reported a while ago.

**Comment by knguyenBC** [comment]
I don't see this as an issue anymore. (iphone 6, build 688)
